src/cljs/chronograph_web/pages/landing/views.cljs

```diff
@@ -1,12 +1,26 @@
 (ns chronograph-web.pages.landing.views
   (:require [re-frame.core :as rf]
+            [chronograph-web.events.organization :as org-events]
             [chronograph-web.subscriptions :as subs]))
 
+(defn organizations-list-element [slug organization]
+  [:li {:key slug}
+   [:a {:href (str "/organization/" slug)}
+    (:name organization)]])
+
+(defn organizations-list [organizations]
+  [:div
+   (when (not-empty organizations)
+     [:div
+      [:p "Select organization"]
+      [:ul (map #(apply organizations-list-element %) organizations)]])
+   [:a {:href "/organizations/new"} "New Organization"]])
+
 (defn landing-page [_]
-  (let [{:keys [name email photo-url]} @(rf/subscribe [::subs/user-info])]
-    [:div
-     [:a {:href "/organization/new"} "New Organization"]
-     [:h2 "Welcome!"]
-     [:img {:src photo-url}]
-     [:p name]
-     [:p email]]))
+  (fn []
```

Comment:
This should accept the same arguments as the component function, according to the docs.
```suggestion
  (fn [_]
```
That being said, I don't think we need to pass the route params to our pages as props, since they can be looked up in the DB instead. We can make that refactoring later though.

Reply:
Good catch